tag.php

```diff
@@ -34,6 +34,12 @@
 		include 'page.php';
 
 	} else {
+		$view = \P4\MasterTheme\Features::is_active(\P4\MasterTheme\Features::LISTING_PAGE_GRID_VIEW) ? 'grid' : 'list';
+		$query_template = file_get_contents( get_template_directory() . "/parts/query-$view.html" );
+
+		$content = do_blocks( $query_template );
+
+		$context['query_loop'] = $content;
 
```

Comment:
I assume the previous content of the page is no longer relevant, I'll remove it later to simplify the changes in the PR for now.

Reply:
I for now added a feature toggle instead of removing the code.

I did this because I realized that the previous auto generated page has different post types, and the paginated list for now only contains posts.

Another reason was to unblock this PR. The CF changes are still in progress but don't have to block merging everything else because of the feature toggle.